week6/sentimental-readability/readability.py

```diff
@@ -0,0 +1,23 @@
+text = str(input("Text: "))
+
+letters = 0
+words = 1
+sentences = 0
+
+for i in text:
+    if i.isalpha():
+        letters += 1
+    elif i == " ":
+        words += 1
+    elif i == "." or i == "!" or i == "?":
+        sentences += 1
+
+index = 0.0588 * (letters / words * 100) - 0.296 * \
```

Comment:
`index` is a rather generic name, you should name the used index here. 
If a reviewer wants further information on how or why this index is calculated he/she has a term to google. (And can compare wether this implementation of the used index is correct.)

```suggestion
coleman_liau_index = 0.0588 * (letters / words * 100) - 0.296 * \
```